md2fhtml.py

```diff
@@ -0,0 +1,26 @@
+import markdownify
+import markdown
+
+
+def html2mdstr(html_str: str):
+    """Converts html in string form to markdown"""
+    md_str = markdownify.markdownify(html_str)
+    return md_str
+
+
+def html2mdlist(html_list: list):
+    """Converts html as a list of strings to markdown"""
+    html_str = ''.join(html_list)
+    return html2mdstr(html_str)
+
+
+def md2htmlstr(md_str: str):
+    """Converts markdown in string form to html"""
+    html_str = markdown.markdown(md_str)
+    return html_str
+
+
+def md2htmllist(md_list: list):
+    """Converts markdown as a list of strings to html"""
+    md_str = ''.join(md_list)
```

Comment:
shouldn't this be joined with "\n". is the list a list of lines?

Reply:
Thank you for catching this, I believe this is the case. Forgot that a newline seperated string is still a string.